### PR TITLE
feat(dbt-adapter): dbt-athena-community support

### PR DESCRIPTION
**Pull request by chamini2**
Based on #741

### CODE REVIEW

File: .github/workflows/test_integration_adapter.yml [hunk 1]
```diff
@@ -77,11 +78,13 @@ jobs:
...
-            ADAPTER_PACKAGE="dbt-${{ matrix.profile }}"
-          else
-            ADAPTER_PACKAGE="dbt-${{ matrix.profile }}==${{ matrix.dbt_version }}"
+          ADAPTER_PACKAGE="dbt-${{ matrix.profile }}"
+          if [[ '${{ matrix.profile }}' == 'athena' ]]
+            ADAPTER_PACKAGE="dbt-athena-community"
+          fi
+
+          if [[ -n '${{ matrix.dbt_version }}' ]]
+            ADAPTER_PACKAGE="${ADAPTER_PACKAGE}==${{ matrix.dbt_version }}"
```
Comment (mederka) [inline .github/workflows/test_integration_adapter.yml:87]: Tests are running into this error:
```
syntax error near unexpected token `fi'
```

File: projects/adapter/src/dbt/adapters/fal_experimental/utils/environments.py [hunk 2]
```diff
@@ -238,13 +238,20 @@ def _parse_remote_config(
         "target_environments": [env_definition],
     }
 
+def _get_package_from_type(adapter_type: str):
+    SPECIAL_ADAPTERS = {
+        # Documented in dbt website
```
Comment (drochetti) [inline projects/adapter/src/dbt/adapters/fal_experimental/utils/environments.py:243]: nice call-out, maybe also leave the link?

File: .github/workflows/test_integration_adapter.yml (hunk 1 above)
Comment (drochetti) [reply .github/workflows/test_integration_adapter.yml:87]: is it missing the `then`?

File: projects/adapter/src/dbt/adapters/fal_experimental/utils/environments.py (hunk 2 above)
Comment (chamini2) [reply projects/adapter/src/dbt/adapters/fal_experimental/utils/environments.py]: done!

File: projects/adapter/src/dbt/adapters/fal_experimental/support/athena.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+from dbt.adapters.base.relation import BaseRelation
+from dbt.contracts.connection import AdapterResponse
+import sqlalchemy
+import pandas as pd
+from dbt.adapters.base import BaseAdapter
+from urllib.parse import quote_plus
+
+
+def create_engine(adapter: BaseAdapter) -> Any:
+    creds = adapter.config.credentials._db_creds
```
Comment (mederka) [inline projects/adapter/src/dbt/adapters/fal_experimental/support/athena.py:12]: Looks like athena credentials are `adapter.config.credentials`.

Just judging from this error:

```
16:22:06  Unhandled error while executing /tmp/tmpkwcled1j/target/run/source_test/models/model_a.py
'AthenaCredentials' object has no attribute '_db_creds'
Error: 16:22:06.287093 [error] [Thread-1 (]: Unhandled error while executing /tmp/tmpkwcled1j/target/run/source_test/models/model_a.py
'AthenaCredentials' object has no attribute '_db_creds'
16:22:06  1 of 1 ERROR creating python table model ***_custom.model_a ...... [ERROR in 16.62s]
Error: 16:22:06.289416 [error] [Thread-1 (]: 1 of 1 ERROR creating python table model ***_custom.model_a ...... [ERROR in 16.62s]
16:22:06  
16:22:06  Finished running 1 table model in 0 hours 0 minutes and 24.81 seconds (24.81s).
16:22:06  
16:22:06  Completed with 1 error and 0 warnings:
16:22:06  
16:22:06  'AthenaCredentials' object has no attribute '_db_creds'
Error: 16:22:06.302038 [error] [MainThread]: 'AthenaCredentials' object has no attribute '_db_creds'
16:22:06  
```

**Review by bruno-ribeirodasilva — COMMENTED** [review]
Hey, dbt-athena-community now supports dbt 1.4

**Comment by mederka** [comment]
Closing as #809 included these changes

**Pull request closed without merging**